src/pkg/cli/tailAndMonitor.go

```diff
@@ -44,16 +44,18 @@ func TailAndMonitor(ctx context.Context, project *compose.Project, provider clie
 		defer wg.Done()
 		// block on waiting for services to reach target state
 		serviceStates, svcErr = WaitServiceState(svcStatusCtx, provider, targetServiceState, project.Name, tailOptions.Deployment, computeServices)
+		term.Debug("WaitServiceState stopped with", svcErr)
```

Comment:
I don't like printing in Go routine though, it'll be randomly inserted somewhere